packages/contractkit/src/identity/offchain/accessors/name.ts

```diff
@@ -0,0 +1,21 @@
+import * as t from 'io-ts'
+import OffchainDataWrapper from '../../offchain-data-wrapper'
+import { EncryptedSimpleSchema, SimpleSchema } from '../schemas'
+
+const NameSchema = t.type({
+  name: t.string,
+})
+
+export type NameType = t.TypeOf<typeof NameSchema>
+
+export class NameAccessor extends SimpleSchema<NameType> {
+  constructor(readonly wrapper: OffchainDataWrapper) {
+    super(wrapper, NameSchema, '/account/name')
+  }
+}
+
+export class EncryptedNameAccessor extends EncryptedSimpleSchema<NameType> {
```

Comment:
Instead of `EncryptedNameSchema` and `NameSchema`, I would suggest `PrivateNameSchema` and `PublicNameSchema` or something along those lines. In particular, calling out that the unencrypted version is **public**. We want to make sure folks don't accidentally publicly attach their name to their account if they do not mean to. This is also from the fact that, at least in the applications we are creating immediately, encryption is the default rather than being unusual. Same comment applies to the profile picture schema.